request_handlers.h

```diff
@@ -0,0 +1,13 @@
+#ifndef REQUEST_HANDLER_H
+#define REQUEST_HANDLER_H
+
+#include <boost/asio.hpp>
+namespace bai = boost::asio::ip;
```

Comment:
Addressed as part of https://github.com/UCLA-CS130/Mr.-Robot-et-al./pull/15/commits/83b15ef868d5dc7f6c0b4da4df3f155f05d6af12